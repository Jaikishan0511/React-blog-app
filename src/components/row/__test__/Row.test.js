import React from "react";
import renderer from "react-test-renderer";

import Row from "../index";
describe("Row", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<Row />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
