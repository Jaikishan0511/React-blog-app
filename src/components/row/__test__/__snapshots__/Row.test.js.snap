// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row snapshot renders 1`] = `
<div
  className="row"
>
  <div
    className="cell"
    id="cell"
  />
  <div
    className="cell"
    id="cell"
  />
  <div
    className="cell"
    id="cell"
  />
</div>
`;
